### PR TITLE
Mile stone contact form

### PR DESCRIPTION
**Pull request by Saadat123456**
**Portfolio: add a contact form and finish the mobile version**

### Project requirements

- The section you need to build for this milestone is: 
    - the contact form in the footer:
We included a screenshot of Template 1 for reference, but you should follow the template that you chose.
- Make sure that:
    - Name field accepts up to 30 chars.
    - Email field accepts only emails.
    - Message field accepts up to 500 chars.
    - All fields have placeholder text.
    - All fields have the correct type.
    - A user cannot submit the form if one or more of the fields is empty.
- Collect data by using the [Formspree service](https://formspree.io/html).
- In order to lay out the elements on the page you should use Flexbox.
- You don't need to implement any functionality that requires JavaScript.
- You must stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
- Implement the button interactions (enable, hover, etc.).
- **You need to use pair-programming for this project. We will check the commit history to make sure that both partners have contributed code on each project. Therefore you need to implement the above features in all of the portfolios in your pair-programming group.**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,188 +1,267 @@
...
+              id="msg"
+              placeholder="Enter your message"
+              maxlength="500"
+              required
+            ></textarea>
+          </li>
+          <li>
+            <button type="submit">Get in touch</button>
+          </li>
+        </ul>
```
Comment (mateo951) [inline index.html:235]: Please try to follow the details on the figma's design 📝 Your inputs for name, email, and textarea have a red border. What do you think if we change those borders to meet the project requirements 🙌😎

#### Your design 
image

#### Figma's design 
image

File: index.html [hunk 2]
```diff
@@ -1,188 +1,267 @@
...
+          </ul>
+        </li>
+      </ul>
+    </section>
+    <section class="footer">
+      <div class="form-desc">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </div>
+      <form
```
Comment (mateo951) [inline index.html:200]: Amazing job with setting up formspree 🔥 Working like a charm!

**Review by mateo951 — CHANGES_REQUESTED** [review]
Hi @Saadat123456 and @karam084,

📝👌 **Your project highlights** 🔥😎 
- Linters are on point ✔
- Nice README file 👀
- Splendid PR 👌😎
- UI design 🔥

You both have done a great job so far 🔥
There is one more issue that you still need to address to go to the next project but you are almost there 😁

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Review by Rustamxon7 — APPROVED** [review]
## Hi team👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

image

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**